dapr/proto/runtime/v1/appcallback.proto

```diff
@@ -107,6 +114,63 @@ message TopicEventResponse {
   TopicEventResponseStatus status = 1;
 }
 
+// TopicEventBulkRequestEntry represents a single message inside a bulk request
+message TopicEventBulkRequestEntry {
+  // Unique identifier for the message.
+  string entryID = 1;
+
+  // The content of the event.
+  bytes event = 2;
+  
+  // content type of the event contained.
+  string content_type = 3;
+  
+  // The metadata associated with the event.
+  map<string,string> metadata = 4;
+}
+
+// TopicEventBulkRequest represents request for bulk message
+message TopicEventBulkRequest {
+    // Unique identifier for the bulk request.
+  string id = 1;
+
+  // The list of items inside this bulk request.
+  repeated TopicEventBulkRequestEntry entries = 2;
+
+  // The metadata associated with the this bulk request.
+  map<string,string> metadata = 3;
+
+  // The pubsub topic which publisher sent to.
+  string topic = 4;
+    
+  // The name of the pubsub the publisher sent to.
+  string pubsub_name = 5;
+  
+  // The type of event related to the originating occurrence. 
+  string type = 6;
+
+  // The matching path from TopicSubscription/routes (if specified) for this event.
+  // This value is used by OnTopicEvent to "switch" inside the handler.
+  string path = 7;
+}
+
+// TopicEventBulkResponseEntry Represents single response, as part of TopicEventBulkResponse, to be
+// sent by subscibed App for the corresponding single message during bulk subscribe
+message TopicEventBulkResponseEntry {
+  // Unique identifier associated the message.
+  string entryID = 1;
```

Comment:
here too, per naming conventions should be `entry_id`

Reply:
Changing to `entry_id` from `entryID` in proto definition. The generated code is now `EntryId`.
On the other hand, due to linting, the structs that we generate internally ourselves can have `EntryID` only.
It looks very odd and confusing in code to have `something.EntryID = otherthing.EntryId` or vice versa. 
Since, proto definitions are outside world facing, so adhering to that nomenclature only, and for internal structs would try to use `EntryId` only and skip linting.
Hope that seems fine.

Reply:
Raised https://github.com/dapr/components-contrib/pull/2141 and https://github.com/dapr/dapr/pull/5285 for this.